src/options.cpp

```diff
@@ -761,13 +761,6 @@ void initOptions()
                                  );
 
     ////////////////////////////GRAPHICS/////////////////////////
-    OPTIONS["NO_BRIGHT_BACKGROUNDS"] = cOpt("graphics", _("No bright backgrounds"),
-                                            _("If true, bright backgrounds are not used - some consoles are not compatible."),
-                                            false
```

Comment:
OK, that's good to know.  I'm likely to have to run today but hope to get the net up in Pittsburgh, so hopefully will be able to merge again there tonight.

(Otherwise, more equine-mutcat work, so it's not all bad.)